src/content/docs/en/pages/devtools/api-graphql/features/features.mdx

```diff
@@ -283,7 +283,9 @@ Using pagination can be useful when you get a large number of data in response t
 
 The **offsets** parameter sets the number of records you want to skip in your data response, and the **limits** parameter sets the number of results you want to receive.
 
-> Setting offset and limit parameters isn't mandatory. In case you don't set the parameters, GraphQL API automatically sets the **offset** for **0** and the **limit** for **10**.
+:::note
+Setting offset and limit parameters isn't mandatory. In case you don't set the parameters, GraphQL API automatically sets the **offset** for **0** and the **limit** for **10**.
```

Comment:
```suggestion
Setting offset and limit parameters isn't mandatory. In case you don't set the parameters, GraphQL API automatically sets the **offset** for `0` and the **limit** for `10`.
```